boards/nrf51dk/src/main.rs

```diff
@@ -126,6 +127,7 @@ impl kernel::Platform for Platform {
             8 => f(Some(self.led)),
             9 => f(Some(self.button)),
             14 => f(Some(self.rng)),
+            34 => f(Some(self.aes)),
```

Comment:
AES is a generaly purpose driver (although only supported by the nRF as of this PR), which we generally want to use low-numbered driver numbers for and also document in `docs/Syscalls.md` (at the bottom under "Allocated Driver Numbers"). Looks like 17 is next available one.

_*At some point we'll go through and order these in some consistent and useful way, but for now, just gran the next available one_